Countries/ListViewController.swift

```diff
@@ -66,6 +59,18 @@ class ListViewController: UITableViewController {
     override func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
         return TableConstants.heightForRow
     }
+    
+    override func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        guard let header = tableView.dequeueReusableHeaderFooterView(withIdentifier: CountryTableViewHeader.identifier) as? CountryTableViewHeader else { return UITableViewHeaderFooterView() }
+        let earthImage = UIImage(named: TableConstants.headerEarthImageName)
+        header.earthImageView.image = earthImage
+        header.label.text = TableConstants.headerLabelText
```

Comment:
If all (two) properties are constants can we move them into `CountryTableViewHeader`?

Reply:
shure, there is no reason to keep them outside CountryTableViewHeader..